client/state/ui/editor/reducer.js

```diff
@@ -63,6 +64,15 @@ export function isLoading( state = false, action ) {
 	return state;
 }
 
+export function isLoaded( state = false, action ) {
+	switch ( action.type ) {
+		case EDITOR_LOADED:
+			return get( action, 'isLoaded', false );
```

Comment:
If you want `EDITOR_RESET` to reset `isLoaded` to `false`, this reducer needs to react to the `EDITOR_RESET` action, too.

Reply:
Refactored so we're not using `EDITOR_RESET` in 898ca2f